api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -262,7 +262,7 @@ public CohortDefinition getPatientsWhoAreOnTarget() {
         EptsReportUtils.map(
             getPatientsWhoAreOnRoutine(),
             "startDate=${startDate},endDate=${endDate},location=${location}"));
-    cd.setCompositionString("results AND NOT routine");
+    cd.setCompositionString("results AND5NOT routine");
```

Comment:
This one seems to be an inadvertent change. Please correct

Reply:
Yap, good catch, that is NOT supposed  to be changed in this ticket

Reply:
Those fixed.